pkg/reconciler/pipelinerun/timeout_test.go

```diff
@@ -33,135 +33,14 @@ import (
 func TestTimeoutPipelineRun(t *testing.T) {
 	testCases := []struct {
 		name                  string
-		embeddedStatus        string
 		useV1Beta1CustomTasks bool
 		pipelineRun           *v1beta1.PipelineRun
 		taskRuns              []*v1beta1.TaskRun
 		customRuns            []*v1beta1.CustomRun
 		runs                  []*v1alpha1.Run
 		wantErr               bool
 	}{{
```

Comment:
Same comment as for cancellation tests, I'm not sure all of these test cases can be safely removed